### ISSUE TITLE
Document how to Retrain Model from Scratch if Technically Possible

### ISSUE THREAD

**Issue by DarioS** [MouseLand/cellpose#1446]
> You should only start training with the built-in cpsam model

I would like to know how to train a 12-channel model from scratch, e.g. HH3, panCK, CD20, CD45, FoxP3, CD3, CD4, CD8, CD14, CD31, CD56, CD66. I want to be able to separate touching cells that are incorrectly reported as one cell when so many cytoplasmic markers are combined into one value by a maximum summary.

<img width="1046" height="629" alt="Image" src="https://github.com/user-attachments/assets/e267df84-3b48-49d8-bcfe-4f80485bfcbb" />

**Comment by carsen-stringer** [comment]
The learning rate for retraining is available in the paper, 12-channels would require modification of the patch embeddings layer, you could for example replicate it or try other approaches, we do not support that here
